### PR TITLE
i18n(zh-cn): Translate `guides/deploy/aws.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

Translate `guides/deploy/aws.mdx` content to Chinese.

### CODE REVIEW

File: src/content/docs/zh-cn/guides/deploy/aws.mdx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    :::tip
+    存储桶名称应该是全局唯一的，我们建议使用你的项目名称以及你的站点域名来组合。
+    :::
+
+2. 禁用 **“Block all public access”**。默认情况下，AWS 将所有存储桶设置为私有。要使其公开访问，你需要在存储桶的属性中取消选中 “Block public access” 复选框。
+
+3. 将你构建后的文件（位于 `dist` 目录）上传到 S3。你可以在控制台手动执行此操作，也可以使用 AWS CLI。如果使用 AWS CLI，请在 [使用 AWS 凭证进行身份验证](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) 后，使用以下命令：
+
+    ```
+    aws s3 cp dist/ s3://<存储桶名称>/ --recursive
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/deploy/aws.mdx:80]:
```suggestion
    aws s3 cp dist/ s3://<BUCKET_NAME>/ --recursive
```
这里应该跟下面的警告提示对应上

File: src/content/docs/zh-cn/guides/deploy/aws.mdx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    ```
+
+    :::caution
+    请确保将 `<CLOUDFRONT_OAI_ID>` 替换为你的 CloudFront Origin Access Identity ID 的名称。你可以在设置 CloudFront 后，在 **CloudFront > Origin access identities** 中找到 CloudFront Origin Access Identity ID。
+    :::
+
+### CloudFront 设置
+
+1. 创建一个 CloudFront 分发，并按如下值配置：
+    * **源域名：**你的 S3 存储桶
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/deploy/aws.mdx:153]:
```suggestion
    * **源域名：** 你的 S3 存储桶
```
这里格式渲染错误了，看一下

File: src/content/docs/zh-cn/guides/deploy/aws.mdx [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+    * **S3 存储桶访问：**"Yes use OAI (bucket can restrict access to only CloudFront)"
+    * **源访问标识：**创建一个新的源访问标识
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/deploy/aws.mdx:155]:
```suggestion
    * **源访问标识：** 创建一个新的源访问标识
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good job peeps 🚀

**Pull request merged**